src/cancelContentPlay.js

```diff
@@ -37,6 +37,16 @@ const cancelContentPlay = function(player) {
       player.el_.style.display = 'block';
       placeholder.remove();
     });
+
+    // Detect fullscreen change, remove placeholder and show player.
+    // On iOS 10 Safari is supposed to also add playsinline to the player
```

Comment:
For correctness, I'm mentioning it here, not sure whether we need to update the player. iOS 10 would only add `playsinline` if you pinch to exit or if you hit the exit fullscreen button on the bottom right side of the player. Hitting "done" should not be adding this property.
Also, it adds the `controls` attribute in addition to `playsinline`, we might need to account for that as well.